interfaces/cython/cantera/ctml_writer.py

```diff
@@ -527,6 +528,7 @@ def __init__(self,
                  note = '',
                  thermo = None,
                  transport = None,
+                 standardState = None,
```

Comment:
Is there a reason to add this argument here, rather than as the last argument? If it is added here, creating any instances of this class have to modify the order of their arguments if they're not using keyword arguments.

Reply:
I'm not sure if there's a reason, but I'm hard pressed to think of a case when someone is going to attempt to create a species from input (cti, xml, or yaml) without using any keywords...

That said, i don't really have a problem with moving it.  Hopefully @manikmayur can comment

Reply:
I agree that the new argument needs to be last.

Reply:
I agree too, it can be moved to the end.

Reply:
DONE.